config/jobs/kubernetes/sig-scalability/sig-scalability-experimental-periodic-jobs.yaml

```diff
@@ -196,7 +196,7 @@ periodics:
       - --test-cmd=$GOPATH/src/k8s.io/perf-tests/run-e2e.sh
       - --test-cmd-args=cluster-loader2
       - --test-cmd-args=--experimental-gcp-snapshot-prometheus-disk=true
-      - --test-cmd-args=--experimental-prometheus-disk-snapshot-name=${JOB_NAME}-${BUILD_ID}
```

Comment:
Ideally we'd change the JOB_NAME to be shorter. Introducing this discrepancy between JOB_NAME and prometheus snapshot name will make some of our internal tools for creating grafana datasources stop working for this job, but I believe this is fine, e.g. not sure if changing the JOB_NAME is worth the hassle.